README.md

```diff
@@ -33,15 +33,18 @@ Demonstrate your understanding of this week's concepts by answering the followin
 Edit this document to include your answers after each question. Make sure to leave a blank line above and below your answer so it is clear and easy to read by your Team Lead
 
 1. If you were to describe semantic HTML to the next cohort of students, what would you say?
-
+    Semantic HTML are tags that contain some sort of meaning. 
```

Comment:
good answer, remember this is important for Search Engines/browsers/accessibility tools